roles/bwc/templates/rbac_roles/roles.yml.j2

```diff
@@ -0,0 +1,6 @@
+---
+
+name: {{ item.name }}
+description: {{ item.description }}
+permission_grants:
+    {{ item.permission_grants | to_nice_json }}
```

Comment:
See https://gist.github.com/lakshmi-kannan/0f5f073311a40d209d3e19c7cab3e864 for why I had to use to_nice_json.